### ISSUE TITLE
Plugin not opening correctly

### ISSUE THREAD

**Issue by KennyDenvers** [micahke/obsidian-gpt3-notes#3]
Hi!
Did some queries, worked well, until at some point only a small piece of the screen keeps coming up when I start to run the plugin. What could be the matter here? Restarted Obsidian several times to no improvement.

![image_56](https://user-images.githubusercontent.com/90445513/211386204-0df6ed3e-372a-4f8d-9efc-fe1ee17f49ed.png)

**Comment by micahke** [comment]
@KennyDenvers is this problem still occurring? If so, let me know what your set up is and I'll try to reproduce.

**Comment by KennyDenvers** [comment]
> @KennyDenvers is this problem still occurring? If so, let me know what your set up is and I'll try to reproduce. 

Yes, still occurring. I figured that the number of tokens (or so) could be limited, per month or something, and that my situation could have to do with that. But not sure about that - I shouldn't be the only who runs out, I guess. And the web-based ChatGPT still works fine for me.

I have the same persisting effect on mobile (iPhone 13, iOS 16.2) and Win11 PC.
